sdk/.prettierrc

```diff
@@ -1,4 +1,11 @@
 {
   "printWidth": 100,
-  "singleQuote": true
-}
\ No newline at end of file
+  "singleQuote": true,
+  "useTabs": false,
+  "arrowParens": "always",
+  "bracketSpacing": true,
+  "endOfLine": "lf",
+  "semi": true,
+  "tabWidth": 2,
+  "trailingComma": "es5"
```

Comment:
It looks like we don't need all those changes to `.prettierrc`, since they're all defaults in Prettier v2 (and almost all are defaults in v1 too). I don’t mind having them explicit either. 🤷

Reply:
I think explicig is better because some users may have these overriden in vscode